lib/atscppapi/src/Transaction.cc

```diff
@@ -98,6 +97,46 @@ Transaction::~Transaction()
   delete state_;
 }
 
+bool Transaction::configIntSet(TSOverridableConfigKey conf, int value) 
+{
+    //txn,conf,value
+    TSReturnCode res=TSHttpTxnConfigIntSet(state_->txn_,conf,(TSMgmtInt)value);
+    return (res==TS_SUCCESS);
```

Comment:
Two comments on all of these methods.
1) `return` is not a function, it does not need parentheses.
2) There's no need for `res` - you can be more succinct with
`return TS_SUCCESS == RSHttpTxnConfigIntSet(...);`